### DISCUSSION TITLE
Showing homepage link

### DISCUSSION THREAD

**Discussion by allprod** [jpanther/congo#95] [Q&A]
I'm new to hugo and am trying to make a blog with this theme but the link to the homepage isn't showing on the top left of the site. I cant see where I'm going wrong and I've gone through the tutorials but cant see anything obvious that I may have missed. link: [page](https:itisentropy.github.io).

I really cant figure out where I am going wrong

**Comment by jpanther** [comment]
Are you trying to use a logo? At first glance it looks like `logo` is uncommented in the config and then you're not supplying an image. I'm only going off your built site though as I can't find a repo for this project.

**Reply by allprod** [reply]
I'm not trying to supply a logo. for now I just want the text to show so I can have a link to homepage but even the text isnt showing. I also noticed in my hugo dev server my post is showing under the blog link (posts page) but on gh pages it doesnt show. The posts page goes back to being empty on live reloads but if i exit and re run the hugo command the posts page populates again. could this be a hugo bug?

**Reply by allprod** [reply]
Also, what is the rule for logos cos the logo is provided like so `logo = "img/logo.jpg"` but doesn't show

**Reply by allprod** [reply]
So.. I half figured our the empty posts page thing. my post seems to be hidden cos it is created as a directory and the `/posts` index isnt linked to it. I dont understand why though. dev server makes the link properly, but running `hugo --minify` on my device has the same problem.

**Reply by jpanther** [reply]
This still sounds like it's a configuration issue. If you only want a text title, then set the `title` parameter and make sure `logo` is commented out. The rules for the logo are explained in the Configuration section of the documentation. The path is relative to the `assets/` folder.

I'm also not sure what you mean about empty posts... that wasn't part of your original question?? This sounds like a content organisation issue though. You should ensure you are familiar with the Hugo content folder structure by taking a look at the official docs at https://gohugo.io/content-management/organization/
